tools/cluster.py

```diff
@@ -372,12 +374,22 @@ def ssh_master(cluster_name, key_file=default_key_file, user=default_remote_user
     ssh_call(user=user, host=master, key_file=key_file, args=args)
 
 
-def exec_shell(cluster_name, command, key_file=default_key_file, user=default_remote_user, region=default_region):
+def exec_shell(cluster_name, command, key_file=default_key_file, user=default_remote_user, region=default_region, sudo=False):
```

Comment:
Maybe we can fire all sub process (in the first for loop), and than wait for all completions (in a seconds for loop).